.buildkite/scripts/steps/checks/saved_objects_definition_change.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+echo --- Check SO definition changes
+cmd="node scripts/jest_integration -u src/core/server/integration_tests/ci_checks"
```

Comment:
These will already be run by the jest integration tests, is it useful to run the test twice?

Reply:
Only running this during the CI check task would be sufficient. Would there be a way to omit this suite when we're running the integration tests on CI?